r_gl.cpp

```diff
@@ -5,674 +5,685 @@
-        .drawMode = drawMode
-    };
-
-    m_PrimitiveDrawCmds.push_back(drawCmd);
+void GLRender::ImDrawTris(Tri *tris, uint32_t numTris, bool cullFace, DrawMode drawMode) {
+	int offset = m_ImPrimitiveBatch->Add(tris, numTris, cullFace, drawMode);
+
+	GLBatchDrawCmd drawCmd = {.offset = offset, .numVerts = 3 * numTris, .cullFace = cullFace, .drawMode = drawMode};
+
+	m_PrimitiveDrawCmds.push_back(drawCmd);
 }
 
-void GLRender::ImDrawTriPlanes(TriPlane* triPlanes, uint32_t numTriPlanes, bool cullFace, DrawMode drawMode)
-{
-    std::vector<Tri> tris;
-    tris.resize(numTriPlanes);
-    for (int i = 0; i < numTriPlanes; i++) {
-        tris[i] = triPlanes[i].tri;
-    }
-    ImDrawTris(tris.data(), numTriPlanes, cullFace, drawMode);
+void GLRender::ImDrawTriPlanes(TriPlane *triPlanes, uint32_t numTriPlanes, bool cullFace, DrawMode drawMode) {
+	std::vector<Tri> tris;
+	tris.resize(numTriPlanes);
+	for (int i = 0; i < numTriPlanes; i++) {
+		tris[i] = triPlanes[i].tri;
+	}
+	ImDrawTris(tris.data(), numTriPlanes, cullFace, drawMode);
 }
 
-GLBatchDrawCmd GLRender::AddTrisToBatch(GLBatch* batch, Tri* tris, uint32_t numTris, bool cullFace, DrawMode drawMode) {
-    int offset = batch->Add(tris, numTris, cullFace, drawMode);
+GLBatchDrawCmd GLRender::AddTrisToBatch(GLBatch *batch, Tri *tris, uint32_t numTris, bool cullFace, DrawMode drawMode) {
+	int offset = batch->Add(tris, numTris, cullFace, drawMode);
 
-    GLBatchDrawCmd drawCmd = {
-        .offset = offset,
-        .numVerts = 3 * numTris,
-        .cullFace = cullFace,
-        .drawMode = drawMode
-    };
+	GLBatchDrawCmd drawCmd = {.offset = offset, .numVerts = 3 * numTris, .cullFace = cullFace, .drawMode = drawMode};
 
-    return drawCmd;
+	return drawCmd;
 }
 
 // Draw triangles with indexed geometry :)
-void GLRender::ImDrawIndexed(Vertex* verts, uint32_t numVerts, uint16_t* indices, uint32_t numIndices, bool cullFace, DrawMode drawMode)
-{
-    int offset = 0;
-    int offsetIndices = 0;
-    if (!m_ImPrimitiveBatchIndexed->Add(verts, numVerts, indices, numIndices, &offset, &offsetIndices, cullFace, drawMode)) {
-        return;
-    }
-
-    GLBatchDrawCmd drawCmd = {        
-        .offset = offset,
-        .indexOffset = offsetIndices,
-        .numVerts = numVerts,
-        .numIndices = numIndices,
-        .cullFace = cullFace,
-        .drawMode = drawMode
-    };
-
-    m_PrimitiveIndexdDrawCmds.push_back(drawCmd);
+void GLRender::ImDrawIndexed(Vertex *verts, uint32_t numVerts, uint16_t *indices, uint32_t numIndices, bool cullFace,
+							 DrawMode drawMode) {
+	int offset = 0;
+	int offsetIndices = 0;
+	if (!m_ImPrimitiveBatchIndexed->Add(verts, numVerts, indices, numIndices, &offset, &offsetIndices, cullFace,
+										drawMode)) {
+		return;
+	}
+
+	GLBatchDrawCmd drawCmd = {.offset = offset,
+							  .indexOffset = offsetIndices,
+							  .numVerts = numVerts,
+							  .numIndices = numIndices,
+							  .cullFace = cullFace,
+							  .drawMode = drawMode};
```

Comment:
It kept this multiline, but inlined with the braces?

Reply:
I think this should never happen. Either write everything inline, or every statement on its own line (both should be allowed).